arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -1,6 +1,10 @@
+# pylint: disable=no-member, invalid-name, redefined-outer-name
 import typing as tp
 import numpy as np
 import pytest
+import pyjags
+
+import arviz as az
```

Comment:
we try to use `from arviz import from_pyjags` style imports

Reply:
Ok, I made the change. Please let me know if this is not what you had in mind!